src/pages/index.js

```diff
@@ -35,10 +41,14 @@ const App = ({ isSuccessfulSearch, searchResults }) => {
 App.propTypes = {
     isSuccessfulSearch: PropTypes.bool,
     searchResults: PropTypes.arrayOf(COURSE_MODEL),
+    searchTerm: PropTypes.string,
 };
 
-const getServerSideProps = async () => {
-    const courseSearchUrl = `${process.env.COURSES_API_BASEURL}?search=maths`;
+const getServerSideProps = async (context) => {
+    const searchTerm = context.query.search ? context.query.search : "maths";
```

Comment:
I miss groovy ?:

Reply:
This could be hacked together with 
```suggestion
    const searchTerm = context.query.search || "maths";
```
I think, it would take the first option if truthy or the second if not.